src/content/posts/2022/bootstrap-5-2-1.md

```diff
@@ -0,0 +1,43 @@
+---
+author: mdo
+date: "2022-09-02T16:00:00Z"
+title: Bootstrap 5.2.1
+video:
+keywords:
+  - bootstrap
+  - release
+---
+
+Bootstrap v5.2.1 is here with fixes from our latest minor release, v5.2. As a patch release, these changes are limited to bug fixes, documentation updates, and some dependency updates.
+
+## Highlights
+
+- Added a `transparent` default hover border color CSS variable for buttons to fix a visual regression
+- Horizontal list groups with only one child now render the correct `border-radius`
+- Input groups have updated `z-index` values to ensure proper rendering of validated form fields
+- Modified the `list-group-item` selectors to better support nested imports of Bootstrap's CSS
+- Floating labels now reset their `text-align` to ensure consistent styling
+- Updated Autoprefixer to fix warnings of the `color-adjust` property (thanks to @julien-deramond on our team for reporting an issue upstream here)
+- Fixed incorrect `border-radius` values inside pagination components
+- Moved our search functionality out of the hidden sidebar on narrow viewports and mobile devices
+- Removed links to and mentions of Slack from across the codebase, as we intend to shutter Slack in favor of GitHub Discussions
```

Comment:
Can maybe grouped together with the last one:
> - Removed links to Slack from our docs as we've closed it down in favor of GitHub Discussions

Since it has an impact on how the users can communicate with us and with each other, maybe it deserves a separate part/paragraph in this article?